tests/test_freshdesk_start_date.py

```diff
@@ -14,6 +14,21 @@ def name():
         return "tap_tester_freshdesk_start_date_test"
 
     def test_run(self):
+        # Streams to verify start date tests
+        expected_streams = self.expected_streams()
+
+        # To collect "time_entries", "satisfaction_ratings" pro account is needed. Skipping them for now.
+        expected_streams = expected_streams - {'satisfaction_ratings', 'time_entries'}
```

Comment:
Can we merge these two separate lines into one?

```python
expected_streams = self.expected_streams() - {'satisfaction_ratings', 'time_entries'}
```

Reply:
Merged mentioned two lines to reduce redundancy.